additionalCSS/aboutMe.css

```diff
@@ -0,0 +1,88 @@
+.aboutMe {
+  width: 100%;
+  align-items: center;
+}
```

Comment:
- [ ] As suggested by the previous reviewer please use the property `display: flex` on your `.aboutMe` class as according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) you must align the content inside your about me section using `flexbox`

<hr>

>  - Make sure the content of the "About myself" section is aligned with `flexbox`

<hr>

_Note: You are using the property `align-items: center;` inside `.aboutMe ` but if check it using the chrome development tool you will see that this property is not taking any effect. This is because `align-items: center;` is a property of `flexbox`. By default a `section` tag is `block` and `align-items` property does not work on `block` items_